content/chat/rooms/presence.textile

```diff
@@ -89,53 +129,128 @@ Users can enter and leave the presence set of a room to indicate when they are o
 
 ```[javascript]
 await room.presence.update({ status: 'busy' });
 ```
 
-Use the "@presence.leave()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_js.Presence.html#leave method to explicitly remove a user from the presence set. This will send a presence event to all users subscribed to presence. You can also set an optional data field such as setting a status of 'Back later'.
+```[react]
+import React from 'react';
+import { usePresence } from '@ably/chat/react';
+
+const MyComponent = () => {
+  const { update, isPresent } = usePresence({
+    dataToEnterWith: { status: 'Online' },
+  });
+
+  const updatePresence = () => {
+    update({ status: 'Away' });
+  };
+
+  return (
+    <div>
+      <div>Presence status: {isPresent ? 'Online' : 'Offline'}</div>
+      <button onClick={updatePresence}>Set Away</button>
+    </div>
+  );
+};
+```
 
-When a user goes offline or closes their "connection":/chat/connect, a leave event is also emitted and they are removed from the presence set.
+blang[javascript].
+  Use the "@presence.leave()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_js.Presence.html#leave method to explicitly remove a user from the presence set. This will send a presence event to all users subscribed to presence. You can also set an optional data field such as setting a status of 'Back later'.
+
+blang[react].
+  Indicate when a user leaves a room with the "@usePresence@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/functions/chat_react.usePresence.html hook. Users will automatically be removed from the presence set when the component unmounts.
 
 ```[javascript]
 await room.presence.leave({ status: 'Be back later!' });
 ```
 
+```[react]
+import React from 'react';
+import { usePresence } from '@ably/chat/react';
+
+const MyComponent = () => {
+  const { leave, isPresent } = usePresence({
+    dataToLeaveWith: { status: 'Offline' },
+  });
+
+  return (
+    <div>
+      <div>Presence status: {isPresent ? 'Online' : 'Offline'}</div>
+    </div>
+  );
+};
+```
+
+When a user goes offline or closes their "connection":/chat/connect, a leave event is also emitted and they are removed from the presence set.
+
 h2(#options). Presence options
 
 When "creating a room":/chat/rooms#create there are two options that can be set for the presence feature:
 
 |_. Property |_. Description |_. Default |
 | enter | Set whether the client has permissions to enter the presence set. Calling @presence.enter()@ is still required. | true |
-| subscribe | Set whether the client has permissions to subscribe to the presence set. Calling @presebce.subscribe()@ is still required. | true |
+| subscribe | Set whether the client has permissions to subscribe to the presence set. Calling @presence.subscribe()@ is still required. | true |
```

Comment:
Calling `presence.enter` etc isn't required if you're using React, we do that for you - so probably should hide the sentence here?